### PR TITLE
Feature/josh/about us

### PR DESCRIPTION
**Pull request by joshuaobinna**

### CODE REVIEW

File: src/components/aboutUs/components/TeamProfile.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+class TeamProfile extends React.Component {
+    render() {
+        return(
+            <List  
+                bordered={false}
+                dataSource={TEAM_INFO}
+                grid={{ gutter: 12, lg: 4, md: 4, sm: 1.8, xs: 1.8, xxl: 3 }}
+                renderItem={item => (
+                    <List.Item>
+                        <div key={item.name}>
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/TeamProfile.js]: no need to have this `div` add the key to the `card` component

File: src/components/aboutUs/components/constants.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        FADE: 'fade',
+        GITHUB: 'github',
+        ICON: 'icon',
+        LINKEDIN: 'linkedin',
+        SQUARE: 'square',
+    },
+    TEXTS: {
+        HEADER_TEXT: 'The Team',
+        MID_TEXT: 
+        ` At Agroeasy, we believe there is a better way for you to  market your product `,
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/constants.js]: move line 23 to start on line 22

File: src/components/aboutUs/components/TeamProfile.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                            <Card
+                                hoverable
+                                bordered={false}
+                                className={TEAM_CARD}
+                                cover={
+                                    <Avatar src={item.photo} shape={SQUARE} className={AVATAR} />
+                                }
+                                actions={[<Icon type={LINKEDIN} key={ICON} />, 
+                                    <Icon type={FACEBOOK} key={ICON} />, 
+                                    <Icon type={GITHUB} key={ICON} />]}
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/TeamProfile.js]:
```
 actions={[
    <Icon type={LINKEDIN} key={ICON} />, 
    <Icon type={FACEBOOK} key={ICON} />, 
    <Icon type={GITHUB} key={ICON} />
]}

File: src/components/aboutUs/components/AboutUs.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+                <div className={TOP_DIV}>
+                    <Card bordered={false} className={TOP_CARD}>
+                        <p className={TOP_TEXT}>{TOPMOST_TEXT}</p>
+                    </Card>
+                </div>
+                <div>
+                    <Carousel autoplay effect={FADE}>
+                        {
+                            MARKET_SCENES.map(market_scene => (
+                                <div key={market_scene}>
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/AboutUs.js]: you do not need this `div` just return the `img`

File: src/components/aboutUs/components/AboutUs.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+                                    <img src={market_scene} width={width} />
+                                </div>
+                            ))
+                        }
+                        
+                    </Carousel>
+                </div>
+                <div>
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/AboutUs.js]: this `div` servers no purpose

File: src/components/aboutUs/components/AboutUs.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+                    <h1 className={TEAM_HEADER}><u>{HEADER_TEXT}</u></h1>
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/AboutUs.js]: delete this `h1`, use the `List.header` to display the title for the cards

File: src/components/aboutUs/components/TeamProfile.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    STRINGS: { FACEBOOK, GITHUB, ICON, LINKEDIN, SQUARE }, 
+} = ABOUT_US;
+
+class TeamProfile extends React.Component {
+    render() {
+        return(
+            <List  
+                bordered={false}
+                dataSource={TEAM_INFO}
+                grid={{ gutter: 12, lg: 4, md: 4, sm: 1.8, xs: 1.8, xxl: 3 }}
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/TeamProfile.js]: why 1.8 for sm and xs?

File: src/components/aboutUs/components/TeamProfile.js [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+                                }
+                                actions={[<Icon type={LINKEDIN} key={ICON} />, 
+                                    <Icon type={FACEBOOK} key={ICON} />, 
+                                    <Icon type={GITHUB} key={ICON} />]}
+                            >
+                                <Meta
+                                    title={item.name}
+                                    description={<p className={TEAM_PROFILE} >
+                                        {item.description}
+                                    </p>}
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/TeamProfile.js]: clean up code
```
description={
    <p className={TEAM_PROFILE} >
        {item.description}
    </p>
}
```

File: src/components/aboutUs/components/constants.js [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+    },
+];
+
+export const MARKET_SCENES = [
+    '/images/rfdtfa.jpg',
+    '/images/oilmill.jpg',
+    '/images/marketer.jpg',
+    '/images/market.jpg',
+    '/images/fishseller.jpg',
+];
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/constants.js]: this is a proper syntax for an array with multiple items that can't fit on one line. Can you look through your code and make the other array definitions match this syntax/pattern?

**Comment by Way2nnadi** [comment]
@joshuaobinna The cards are not all the same height. This is quite jarring and undesirable, put in the fix. 
screen shot 2019-01-01 at 4 46 21 pm

File: src/components/aboutUs/components/AboutUs.js [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+import { ABOUT_US, IMAGE_STYLE, MARKET_SCENES } from './constants';
+import TeamProfile from './TeamProfile';
+
+const { 
+    CLASSNAMES: { CONTENT, TEAM_HEADER, TOP_CARD, TOP_DIV, TOP_TEXT },
+    STRINGS: { FADE },
+    TEXTS: { HEADER_TEXT, TOPMOST_TEXT }, 
+} = ABOUT_US;
+const { width } = IMAGE_STYLE;
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/AboutUs.js]: `width` should be `WIDTH` given that its a constant variable

File: src/components/aboutUs/components/constants.js [hunk 11]
```diff
@@ -0,0 +1,86 @@
+export const ABOUT_US = {
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/constants.js]: move this file to the path `src/components/aboutUs/constants.js`

File: src/components/aboutUs/components/AboutUs.js [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+class AboutUs extends React.Component {
+    render() {
+        return (
+            <div className={CONTENT}>
+                <div className={TOP_DIV}>
+                    <Card bordered={false} className={TOP_CARD}>
+                        <p className={TOP_TEXT}>{TOPMOST_TEXT}</p>
+                    </Card>
+                </div>
+                <div>
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/AboutUs.js]: unneeded `div` around `Carousel` component

File: src/components/aboutUs/components/TeamProfile.js (hunk 8 above)
Comment (Way2nnadi) [reply src/components/aboutUs/components/TeamProfile.js]: actually remove the `p` tag around `item.description`. Allow the card element to style the description content

File: src/components/aboutUs/components/TeamProfile.js (hunk 1 above)
Comment (joshuaobinna) [reply src/components/aboutUs/components/TeamProfile.js]: ok

File: src/components/aboutUs/components/constants.js [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+        LINKEDIN: 'linkedin',
+        SQUARE: 'square',
+    },
+    TEXTS: {
+        HEADER_TEXT: 'The Team',
+        MID_TEXT: 
+        ` At Agroeasy, we believe there is a better way for you to  market your product `,
+
+        TOPMOST_TEXT:
+        `At Agroeasy, we all come to work everyday because we want to solve one of the major
```
[the remark this answers is not in the document]
Comment (joshuaobinna) [reply src/components/aboutUs/components/constants.js]: done

File: src/components/aboutUs/components/constants.js [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+        istribution of products between producers, marketers and consumers easy.We want to make
+        data / information available to as mamy as possible. `,
+
+    },
+};
+
+export const TEAM_INFO = [
+    {
+        description: `
+            Front-end developer at Agroeasy, he's a graduate of Federal university
```
[the remark this answers is not in the document]
Comment (joshuaobinna) [reply src/components/aboutUs/components/constants.js]: ok

File: src/components/aboutUs/components/constants.js [hunk 15]
```diff
@@ -0,0 +1,86 @@
...
+            to software engineering. His carear as a software engineer started at Enye (2017),
+            where he became aware of numerous challenges that could be solved via the tech.
+            At Enye, he learnt how best to be able to accomplish. His focus and passion will 
+            ensure Agroeasy provides the best of service to all our customers. `,
+        name: "Josh Obinna",
+        photo: '/images/obinna.jpg',
+    },
+    {
+        description: `
+            She loves playing around with codes, encountering errors, fixing bugs, reading codes,
```
[the remark this answers is not in the document]
Comment (joshuaobinna) [reply src/components/aboutUs/components/constants.js]: ok

File: src/components/aboutUs/components/styles.css [hunk 16]
```diff
@@ -0,0 +1,85 @@
...
+    animation-fill-mode: forwards;
+    width: 100%;
+    margin-top: -5%;
+    height: 57vh;
+    color: black; 
+    text-align: center;  
+    font-family:  Tahoma,  Verdana, sans-serif;
+   
+}
+
```
[the remark this answers is not in the document]
Comment (joshuaobinna) [reply src/components/aboutUs/components/styles.css]: ok

File: src/App.js [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+                <div className="App">
+                    <Route path='/' exact strict component={Home} />
+                    <Route path='/about/' exact strict component={About} />
+                    <Route path='/profile/' exact strict component={UserProfile} />
+                    <Route path='/who-is-agroeasy/' exact strict component={AboutUs} />
+                </div>
+            </Router>
+        );
+    }
+}
```
[the remark this answers is not in the document]
Comment (joshuaobinna) [reply src/App.js]: ok

File: src/components/aboutUs/components/TeamProfile.js (hunk 3 above)
Comment (joshuaobinna) [reply src/components/aboutUs/components/TeamProfile.js]: OKAY

File: src/components/aboutUs/components/AboutUs.js (hunk 4 above)
Comment (joshuaobinna) [reply src/components/aboutUs/components/AboutUs.js]: done

File: src/components/aboutUs/components/TeamProfile.js (hunk 8 above)
Comment (joshuaobinna) [reply src/components/aboutUs/components/TeamProfile.js]: ok

File: src/components/aboutUs/components/AboutUs.js (hunk 12 above)
Comment (joshuaobinna) [reply src/components/aboutUs/components/AboutUs.js]: OK

File: package.json [hunk 18]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "agroeasy",
-  "version": "1.1.5",
+  "version": "1.1.6",
```
Comment (EngrEric) [inline package.json]: how do we resolve version bumping issue, i increased to same version as you did.
@enyeInc your attention is needed here

Comment (enyeInc) [reply package.json]: Think about it, whoever merges down their code first has the rightful claim to the version bump. The second person would then need to bump their version once more. I will not put anything in place to enforce it, if we agree to that.

File: src/components/aboutUs/components/TeamProfile.js [hunk 19]
```diff
@@ -0,0 +1,48 @@
...
+                    <List.Item>
+                        <div key={item.name}>
+                            <Card
+                                hoverable
+                                bordered={false}
+                                className={TEAM_CARD}
+                                cover={
+                                    <Avatar src={item.photo} shape={SQUARE} className={AVATAR} />
+                                }
+                                actions={[<Icon type={LINKEDIN} key={ICON} />,
```
Comment (EngrEric) [inline src/components/aboutUs/components/TeamProfile.js]: I think mapping is possible here.
```SOCIAL_MEDIA: [
        'facebook',
        'github',
        'linkedin',
    ],```
```const actionList = SOCIAL_MEDIA.map(social => (<Icon type={social} key={ICON} />));```
```actions = { actionList }```

File: package.json (hunk 18 above)
Comment (EngrEric) [reply package.json]: I think this is good for now.

Comment (joshuaobinna) [reply package.json]: if eric pushed first, then he has the right to "version": "1.1.5" 
i will take "version": "1.1.6"
@Simpcyclassy you know what to do

File: src/components/aboutUs/components/TeamProfile.js (hunk 19 above)
Comment (joshuaobinna) [reply src/components/aboutUs/components/TeamProfile.js]: your head dey work @EngrEric

File: package.json (hunk 18 above)
Comment (joshuaobinna) [reply package.json]: we have to avoid future version bump

Comment (Simpcyclassy) [reply package.json]: Yeah I totally agree

File: src/components/app/components/Footer.js [hunk 20]
```diff
@@ -9,7 +10,11 @@ const { CONTAINER, DYNAMIC_FOOTER, ROW } = FOOTER_CLASSNAMES;
 const footers = FOOTER_DATA.map(footer => (
     <div className={DYNAMIC_FOOTER} key={footer.key}>
         <h3>{footer.header}</h3>
-        {footer.items.map(item => <div key={item.title}>{item.title}</div>)}
+        {footer.items.map(item => (<div key={item.title}>
+            {
+                item.link ?  <Link to={item.link}>{item.title}</Link> : item.title
+            }
+        </div>))}
```
[the remark this answers is not in the document]
Comment (Simpcyclassy) [reply src/components/app/components/Footer.js]: Looks much easier to read

File: src/components/aboutUs/components/styles.css [hunk 21]
```diff
@@ -0,0 +1,43 @@
...
+    height: 50%;
+    margin-left: 23%;
+    margin-right: 50%;
+    display: block;
+}
+.team-header {
+    text-align: center;
+    font-size: 23px;
+    margin: 2%;
+}
```
Comment (Simpcyclassy) [inline src/components/aboutUs/components/styles.css]: There's additional line at the end of the file

File: src/components/aboutUs/components/TeamProfile.js [hunk 22]
```diff
@@ -0,0 +1,45 @@
...
+                            />
+                        </Card> 
+                    </List.Item>
+                )}
+            />
+        );
+    }
+}
+
+export default TeamProfile;
```
Comment (Simpcyclassy) [inline src/components/aboutUs/components/TeamProfile.js]: @Way2nnadi Is it OK to have two `export default` s in a module like this?

File: src/components/aboutUs/components/styles.css (hunk 21 above)
Comment (joshuaobinna) [reply src/components/aboutUs/components/styles.css]: how do u mean?

File: src/components/aboutUs/components/TeamProfile.js (hunk 22 above)
Comment (Way2nnadi) [reply src/components/aboutUs/components/TeamProfile.js]: you can have multiples `export` s but only one `export default`

File: src/components/aboutUs/components/TeamProfile.js [hunk 23]
```diff
@@ -0,0 +1,45 @@
...
+
+const actionList = SOCIAL_MEDIA.map(social => (<Icon type={social} key={social} />));
+
+class TeamProfile extends React.Component {
+    render() {
+        return(
+            <List  
+                bordered={false}
+                dataSource={TEAM_INFO}
+                grid={{ gutter: 12, lg: 4, md: 1.7, sm: 1.8, xs: 1.7, xxl: 3 }}
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/TeamProfile.js]: Why are you adding `1.7` and `1.8` here?

Comment (joshuaobinna) [reply src/components/aboutUs/components/TeamProfile.js]: responsiveness

File: src/components/aboutUs/components/styles.css (hunk 21 above)
Comment (joshuaobinna) [reply src/components/aboutUs/components/styles.css]: ok

File: src/components/userProfile/components/styles.css [hunk 24]
```diff
@@ -90,6 +90,8 @@
     display: block;
     margin-left: auto;
     margin-right: auto;
+    width: 60%;
```
Comment (Way2nnadi) [inline src/components/userProfile/components/styles.css]: why are you adding this?

Comment (joshuaobinna) [reply src/components/userProfile/components/styles.css]: An error
Sorry

**Comment by Simpcyclassy** [comment]
Looks good to me only that I see two `default export` s at the end of the files in two places `TeamProfile.js` and `AboutUs.js`

**Comment by EngrEric** [comment]
make the links functional, add the links to the various social media of the teams.
Looks good to me

**Comment by joshuaobinna** [comment]
@Way2nnadi do you think I should make the social media links functional just yet?

File: src/components/aboutUs/components/TeamProfile.js [hunk 25]
```diff
@@ -10,24 +10,28 @@ const {
...
                             className={TEAM_CARD}
                             cover={<Avatar src={photo} shape={SQUARE} className={AVATAR} />}
-                            actions={actionList}
+                            actions={
+                                social_media.map(({ icon, link }) => (
+                                    <a href={link} key={icon}>
+                                        <Icon type={icon} />
+                                    </a>
+                                ))
+                            }
```
Comment (Way2nnadi) [inline src/components/aboutUs/components/TeamProfile.js]: Very good job here

**Comment by Way2nnadi** [comment]
@joshuaobinna Overall solid work on here. Go ahead and merge down your code

File: src/components/aboutUs/components/TeamProfile.js (hunk 25 above)
Comment (joshuaobinna) [reply src/components/aboutUs/components/TeamProfile.js]: Thanks

**Pull request merged**